internal/inventory/tagprocessor_test.go

```diff
@@ -23,16 +23,17 @@ func getTestingLogger() logger.LoggingClient {
 	if testing.Verbose() {
 		logLevel = "DEBUG"
 	}
-
-	return logger.NewClientStdOut("test", false, logLevel)
+	mockLogger := logger.NewMockClient()
+	mockLogger.SetLogLevel(logLevel)
```

Comment:
The mock logger doesn't log at all so this has not impact. i.e. not needed.  When verbose is need, return `logger.NewClientStdOut("test", false, logLevel)`

Reply:
Please verify my change is what you were looking for...